types/happypack/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for happypack
+// Project: https://github.com/amireh/happypack
+// Definitions by: Akash Vishwakarma <https://github.com/akashishu777>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+import { Plugin } from 'webpack';
+
+declare class HappyPack extends Plugin {
+
+    constructor(options?: HappyPack.PluginOptions);
+}
+
+declare namespace HappyPack {
+    interface PluginOptions {
+        id?: string;
+        threads?: number;
+        loaders?: any;
+    }
+}
+
+export = HappyPack;
```

Comment:
Some tests to show how this library should be used in TS would be great.

Reply:
Added test.

Reply:
Added test for testing.